okhttp/api/jvm/okhttp.api

```diff
@@ -126,6 +126,10 @@ public abstract interface class okhttp3/Call : java/lang/Cloneable {
 	public abstract fun isCanceled ()Z
 	public abstract fun isExecuted ()Z
 	public abstract fun request ()Lokhttp3/Request;
+	public abstract fun tag (Ljava/lang/Class;)Ljava/lang/Object;
```

Comment:
This seems like a disruptively breaking change. Does the interface default mechanism help?

Reply:
Agreed that it’s disruptive.

The normal use-case of this interface is that it’s implemented by OkHttp, and called by our users. In this normal case there’s no problem, as the implementations include the new functions.

The decorator use case is dangerous: users create their own `Call` implementations that delegate to our implementation, and users’ decorating implementations won’t implement these functions. But I also don’t expect such users to introduce new calls to `tag()` before they re-compile their `Call` implementations.

The most most dangerous case is when a third-party library decorates `Call`, and users introduce new calls to `tag()` without updating such a library. I will document this risk in our changelog!